test/functional/bundleremove/remove-basics.bats

```diff
@@ -55,12 +55,13 @@ global_teardown() {
 	assert_file_exists "$TARGETDIR"/bat/common
 	assert_file_exists "$STATEDIR"/bundles/test-bundle2
 	expected_output=$(cat <<-EOM
+		Removing bundle: test-bundle1
 		Deleting bundle files...
 		Total deleted files: 6
 		Successfully removed 1 bundle
 	EOM
 	)
-	assert_is_output --identical "$expected_output"
+	assert_is_output "$expected_output"
```

Comment:
because I am now printing progress report when deleting the files like this:
```
Removing bundle: test-bundle1

Deleting bundle files...
	...16%
	...33%
	...50%
	...66%
	...83%
	...100%
Total deleted files: 6

Successfully removed 1 bundle
```
And a limitation our test library has is that when comparing percentages in a test we need to use "assert_regex_is_output" or "assert_regex_in_output" because "assert_in_output" will always find  a mismatch with the "tab" in front of the percentage, it cannot detect it.

Reply:
Okay